Very Minor: grammar and quotes (#23265)
mdn/content: files/en-us/web/html/element/input/range/index.md

### step

The `step` attribute is a number that specifies the granularity that the value must adhere to. Only values that match the specified stepping interval ([`min`](#min) if specified, {{htmlattrxref("value", "input")}} otherwise, or an appropriate default value if neither of those is provided) are valid.

> The `step` attribute can also be set to the `any` string value. This `step` value means that no stepping interval is implied and any value is allowed in the specified range (barring other constraints, such as [`min`](#min) and [`max`](#max)). See the [Setting step to the any value](#setting_step_to_the_any_value) example for how this works in supported browsers.

Author: ```suggestion
The `step` attribute can also be set to the `any` string value. This `step` value means that no stepping interval is implied and any value is allowed in the specified range (barring other constraints, such as [`min`](#min) and [`max`](#max)). See the [Setting step to the `any` value](#setting_step_to_the_any_value) example for how this works in supported browsers.
```

> The `step` attribute is a number that specifies the granularity that the value must adhere to, or the special value `any`, which is described below. Only values which are equal to the basis for stepping ([`min`](#min) if specified, {{htmlattrxref("value", "input")}} otherwise, and an appropriate default value if neither of those is provided) are valid.

Reviewer: ```suggestion
The `step` attribute is a number that specifies the granularity that the value must adhere to. Only values that match the specified stepping interval ([`min`](#min) if specified, {{htmlattrxref("value", "input")}} otherwise, or an appropriate default value if neither of those is provided) are valid.
```

> A string value of `any` means that no stepping is implied, and any value is allowed (barring other constraints, such as [`min`](#min) and [`max`](#max)). See the [Setting step to `any`](#setting_step_to_any) example for how this works in supported browsers.

Reviewer: ```suggestion
The `step` attribute can also be set to the `any` string value. This `step` value means that no stepping interval is implied and any value is allowed in the specified range (barring other constraints, such as [`min`](#min) and [`max`](#max)). See the [Setting step to the any value](#setting_step_to_the_any_value) example for how this works in supported browsers.
```
- Dropped the comma before 'and'
- The heading "Setting step to "any"" would also need to be updated. Considering we're not using code styling in headings, I suggest "Setting step to the any value". And then this link here will need to be updated as well.

> > **Note:** When the data entered by the user doesn't adhere to the stepping configuration, the {{Glossary("user agent")}} may round to the nearest valid value, preferring to round numbers up when there are two equally close options.

Reviewer: ```suggestion
> **Note:** When the value entered by a user doesn't adhere to the stepping configuration, the {{Glossary("user agent")}} may round off the value to the nearest valid value, preferring to round numbers up when there are two equally close options.
```
